<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Classroom</class>
 <widget class="QWidget" name="Classroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_previous">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_chalkboard">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>-3</number>
   </property>
   <property name="text">
    <string>What are we going to learn about??</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../src/images/background/Classroom.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_chalkboard_title">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>-3</number>
   </property>
   <property name="text">
    <string>TOPIC</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="archie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>pushButton_back</zorder>
  <zorder>pushButton_previous</zorder>
  <zorder>pushButton_next</zorder>
  <zorder>label_chalkboard_title</zorder>
  <zorder>archie</zorder>
  <zorder>label_chalkboard</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
